.travis.yml

```diff
@@ -29,20 +29,20 @@ jobs:
         - python -m pip install --editable .
         - python -m pytest --cov=ark --pycodestyle ark
     - stage: test_pypi_deploy
-      if: tag IS present
+      # if: tag IS present
       script:
         - "travis_wait 120 sleep 7200 &"
-        - python setup.py bdist_wheel
         - sudo apt-get install patchelf
+        - python setup.py bdist_wheel
         - auditwheel-symbols --manylinux 2_24 dist/*.whl
         - auditwheel repair dist/*.whl --plat manylinux_2_17_x86_64
         - twine upload -r testpypi --skip-existing /home/travis/build/"$TRAVIS_REPO_SLUG"/wheelhouse/*.whl -u "$TEST_PYPI_USERNAME" -p "$TEST_PYPI_PASSWORD"
     - stage: pypi_deploy
       if: tag IS present
```

Comment:
Will you comment/then uncomment this line so that we can have a version of the previous build on pypi?

Reply:
Yeah, should I do the same to the Docker upload as well?

Reply:
Yup, otherwise anyone who tries to run the current start_docker.sh will get an error